build.override.targets

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0"  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--if ServiceUri is set, the service has been already started on another machine-->
+  <Target Name="WcfSetup" Condition="($(WithCategories.Contains('OuterLoop')) or '$(OuterLoop)' == 'true') and ('$(ServiceUri)' == '') and $(OS.Contains('Windows'))" BeforeTargets="BuildAllProjects">
+    <Exec Command="$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\scripts\WCFCustomSetup.cmd" ConsoleToMSBuild="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="WcfSetupOutput" />
+    </Exec>
+    <Message Text="WcfSetupOutput: $(WcfSetupOutput)" />
+  </Target>
+
+  <Target Name="WcfCleanUp" Condition="($(WithCategories.Contains('OuterLoop')) or '$(OuterLoop)' == 'true') and ('$(ServiceUri)' == '') and $(WcfSetupOutput.Contains('WCFServiceCleanupNeeded')) and $(OS.Contains('Windows'))" AfterTargets="Test">
```

Comment:
What if the script failed with error code 1? 

If we want to use the exit code, i'd suggest(-1 = already running, 0=started) and treat both -1 and 0 exit code as successful. Currently anything other than 0 will fail the setup and the build.